### ISSUE TITLE
Some (Containerized?) Applications Always Require Restart

### ISSUE THREAD

**Issue by RayJW** [liske/needrestart#319]
It seems, that since the newly released version 3.8 needrestart always thinks, that apps distributed as a Flatpak or AppImage run outdated binaries. Although, for example, `gnome-shell` I can't explain. But isn't it supposed to list outdated binaries in verbose mode anyway?

This is the output of `sudo needrestart -v` immediately after a full system update and restart. I even tried closing the Filen application, checked that it wasn't listed any more and then once I started it, it was instantly there again.

```shell
[main] eval /etc/needrestart/needrestart.conf
Scanning processes...                                                                                                                                                               
Scanning candidates...                                                                                                                                                              
Scanning processor microcode...                                                                                                                                                     
Scanning linux images...                                                                                                                                                            

Running kernel seems to be up-to-date.

The processor microcode seems to be up-to-date.

No services need to be restarted.

No containers need to be restarted.

User sessions running outdated binaries:
 user @ user manager service: bwrap[2655,3651], filen-desktop[8201,8209], firefox-bin[3661], gnome-shell[2098]

No VM guests are running outdated hypervisor (qemu) binaries on this host. 
```

**Comment by liske** [comment]
This seems to be a duplicate of #317. Feel free to reopen this issue if the fix of #317 does not fix it for you.

**Comment by RayJW** [comment]
> This seems to be a duplicate of #317. Feel free to reopen this issue if the fix of #317 does not fix it for you.

This indeed seems to fix the issue. Running `sudo ./needrestart` doesn't report any outdated processes any more. Just `./needrestart` still lists `bwrap[4148, 7569]` as an outdated process, but I don't think that this is how you're supposed to use `needrestart`, so it might not actually be an issue.
